pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/documentation/ApexDocRule.java

```diff
@@ -133,7 +160,9 @@ private boolean shouldHaveApexDocs(ApexNode<?> node) {
 
         ASTModifierNode modifier = node.getFirstChildOfType(ASTModifierNode.class);
         if (modifier != null) {
-            return (modifier.isPublic() || modifier.isGlobal()) && !modifier.isOverride();
+            Boolean flagPrivate = reportPrivate && modifier.isPrivate();
```

Comment:
The preferred Java type is lowercase `boolean` (`Boolean` is usually only used when generics are involved)

Reply:
Got it, Apex coder habit 😅